docs/fastrtpsgen.rst

```diff
@@ -47,4 +47,4 @@ Where to find *fastrtpsgen*
 
 If you are using the binary distribution of *eProsima Fast RTPS*, *fastrtpsgen* is already provided for you.
 If you are building from sources, you have to compile *fastrtpsgen*.
-You can find instructions in section :ref:`installation-from-sources`.
+You can find instructions in the :ref:`installation`.
```

Comment:
This is not true

Reply:
yes, you're right. It wasn't true anymore, but it will be true, right? We decided to add it to the installation, so it will be true again.
Only problem, we don't have an installation page anymore. This is a little bit of a problem, since there are 4 pages but you do not know to which refer.

Reply:
solution. do not put a reference at all. only Installation.